checkov/common/output/record.py

```diff
@@ -165,7 +167,8 @@ def to_string(self, compact: bool = False, use_bc_ids: bool = False) -> str:
         )
         code_lines = ""
         if self.code_block:
-            code_lines = "\n{}\n".format("".join([self._code_line_string(self.code_block)]))
+            ANSI_COLORS_DISABLED
```

Comment:
@MooreDerek is this line necessary?

Reply:
@nimrodkor The code in `_code_line_string` actually handles the adding of text colour itself - it doesn't use the `colored` function. Happy to refactor to use the same functionality as elsewhere.
https://github.com/bridgecrewio/checkov/blob/master/checkov/common/output/record.py#L124